spec/controllers/bets_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe BetsController, type: :controller do
+  describe 'GET #index' do
+    subject { get :index }
+    it { is_expected.to be_ok }
+  end
+
+  describe 'GET #show' do
+    let(:user) do
+      User.create(username: 'Bob', email: 'bob@mail.com', password: '123')
+    end
+    let(:bet) do
+      Bet.create(bet: 'Bet text',
+                 expires_at: DateTime.tomorrow,
+                 creator_id: user.id)
+    end
+    subject { get :show, params: { id: bet.id } }
+    it { is_expected.to be_ok }
+  end
+
+  describe 'GET #new' do
+    subject { get :new }
+    it { is_expected.to be_ok }
+  end
+
+  describe 'POST #create' do
+    let(:user) do
+      User.create(username: 'Bob', email: 'bob@mail.com', password: '123')
+    end
+    let(:valid_params) do
+      { bet: 'Bet text',
+        expires_at: DateTime.tomorrow,
+        creator_id: user.id }
+    end
+    let(:invalid_params) do
+      { bet: 'Invalid Bet',
+        expires_at: DateTime.yesterday,
+        creator_id: user.id }
+    end
+    subject { post :create, params: { bet: bet_params } }
+    context 'with valid params' do
+      let(:bet_params) { valid_params }
+      let(:created_bet) { Bet.last }
+      it { expect { subject }.to change(Bet, :count).by(1) }
```

Comment:
when you aren't able to write `it { is_expected ... ` in a way that makes sense when reading it back, it is preferrable to write it out on multiplte lines:
```
it 'changes the bet count' do
  expect(subject).to change(etc)
end
```

This is also a situation where naming your subject can helpful
i.e `subject(:get_new) { get :new }` lets you write things like `expect(get_new).to do_something_cool`